iotdb-cli/src/main/java/org/apache/iotdb/cli/client/AbstractClient.java

```diff
@@ -144,6 +145,12 @@
 
   protected static ServerProperties properties = null;
 
+  private static boolean printHeader = false;
+  private static int displayCnt = 0;
+
+  private static final PrintStream SCREEN_PRINTER = new PrintStream(System.out);
+  private static boolean showException = false;
```

Comment:
I did not find any statement to set showException =true, so showException is always false?

Reply:
Yes, as some may think printing stack-trace in console is somehow unfriendly. Maybe we can add this to options in the future if needed.

Reply:
So you can add some comments to indicate `showException` is not used currently and we will add this to options in the future if needed.

Reply:
Added.